best-practices/add_safe_to_evict/resource.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp-pod
+  annotations:
+    cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
+spec:
+  containers:
+  - name: nginx
+    image: nginx:1.14.1
+    volumeMounts:
+    - mountPath: /var/local/aaa
+      name: mydir
+  volumes:
+    - name: mydir
+      emptyDir: {}
+
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp-pod
```

Comment:
This resource is the same name as the above. They must be named uniquely and the names updated in the test manifest.

Reply:
I'll just fix this myself.